entsoe/entsoe.py

```diff
@@ -9,7 +9,7 @@
 import logging
 
 from .exceptions import NoMatchingDataError, PaginationError
-from .mappings import DOMAIN_MAPPINGS, BIDDING_ZONES, TIMEZONE_MAPPINGS, NEIGHBOURS
+from .mappings import DOMAIN_MAPPINGS, TIMEZONE_MAPPINGS, ALL_NEIGHBOURS, ALL_AREA_LOOKUP
```

Comment:
ALL_AREA_LOOKUP is a dictionary that includes BIDDING_ZONES  (which uses the entsoe-py iso2 key system), but also includes CA and BZ string identifiers used more commonly by ENTSOE.